src/app/common/services/project-service.coffee

```diff
@@ -88,6 +88,16 @@ angular.module("doubtfire.common.services.projects", [])
         return task.due_date
       else
         return task.definition.target_date
+    task.days_after_submission = moment().diff(task.submission_date,"days")
+    task.daysAfterSubmission = ->
```

Comment:
Maybe make this `daysAfterSubmissionMessage` and the above one the `daysAfterSubmission`